### ISSUE TITLE
Can not run basic example

### ISSUE THREAD

**Issue by eelcovv** [maresb/ansible-micromamba-role#3]
I tried to run the exmple you give. I have the following playbook:

```
- hosts: 
  -  ubun22orweb
  become: yes
  roles:
    - mambaorg.micromamba
  vars:
    arch: linux-aarch64
    dest: /tmp/micromamba
    root_prefix: /opt/conda
    root_prefix_condarc:
      channels:
        - conda-forge
    packages:
      - mamba
      - python=3.9
```

I have a seperate host file with the conent:

```
all: 
  vars:
    ansible_connection: lxd
    ansible_user: root
    ansible_become: no
  children:
    local:
      vars:
        ansible_lxd_remote: local
      hosts:
        ubun22ordsk: 
        ubun22orweb: 

```

Running the following command fails:

`ansible-playbook -i host.yml  mamba.yml 
`
The error is : 
```

PLAY [ubun22orweb] ********************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************
ok: [ubun22orweb]

TASK [mambaorg.micromamba : Check if micromamba already exists in /tmp/micromamba] ****************************************************
ok: [ubun22orweb]

TASK [mambaorg.micromamba : Make temporary directory for micromamba] ******************************************************************
changed: [ubun22orweb]

TASK [mambaorg.micromamba : Install micromamba] ***************************************************************************************
fatal: [ubun22orweb]: FAILED! => {"changed": false, "msg": "Failed to find handler for \"/root/.ansible/tmp/ansible-tmp-1673189910.9980383-723056-39328482899404/latestbv6vtfan\". Make sure the required command to extract the file is installed. Command \"/usr/bin/unzip\" could not handle archive. Command \"/usr/bin/tar\" could not handle archive."}

PLAY RECAP ****************************************************************************************************************************
ubun22orweb                : ok=3    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

```

Any idea what could be wrong?

**Comment by maresb** [comment]
Thank you very much @eelcovv for the error report! I have not seen this problem before, and I am not yet able to reproduce this.

It seems like this error message can have many possible causes: https://github.com/ansible/ansible/issues/35645

Could you please try again today? Perhaps there was a transient network error. Are you behind any sort of proxy which might be preventing the successful download of the file?

What is the exact operating system and architecture of the target machine?

If the error is still present, then we could try making a separate download step before the extract step. (Ansible might be getting confused by the template variable in `ansible.builtin.unarchive.src`.)

**Comment by eelcovv** [comment]
Hi, I found the problem. I did not have bzip2 installed and the archive is downloaded as a tar.bz2 file. After installing bzip2 it all worked. Thanks!

**Comment by maresb** [comment]
I just released a new version (v1.2.4) which adds a step to install bzip2. Thanks for bringing this to my attention.
